src/handlers/feature_handlers.cpp

```diff
@@ -3565,6 +3565,7 @@ bool playerInPursuit = false;
 bool playerInCooldown = false;
 bool enableEscapeTimer = false;
 bool invertEscapeTimer = false;
+bool collide = false;
```

Comment:
Thi is a very generic name for a global variable, and clashes with the naming of some local variables.

Reply:
I don't have a name in my mind currently but I will try pick another one.